react/features/conference/components/styles.js

```diff
@@ -35,5 +35,11 @@ export default createStyleSheet({
         // On iPhone X there is the notch. In the two cases BoxModel.margin is
         // not enough.
         top: BoxModel.margin * 3
+    },
+
+    videoQualityLabel: {
+        right: 0,
+        top: 0,
+        position: 'absolute'
```

Comment:
Does the label display at the very edge of the screen or is there margin/padding somewhere that brings it away from the edge?

Reply:
In react native, 'absolute' is not really absolute. There is a significant margin that is added by the parent Conference element (see screenshot above).